base/qaqc/DESCRIPTION

```diff
@@ -12,10 +12,13 @@ Author: David LeBauer, Tess McCabe
 Maintainer: David LeBauer <dlebauer@illinois.edu>
 Description: PEcAn integration and model skill testing
 Imports:
+    dplyr,
     graphics,
+    rlang,
```

Comment:
Nit: we prefer to list imports in alphabetical order (R doesn't care, this just makes it easier to skim when asking "Let's see, does this package depend on ____?")

```suggestion
```